src/solver/solvers.F90

```diff
@@ -1014,6 +1014,9 @@ subroutine solveState
     ! call, "Iter" and "Iter Total" will both display 0.
     approxTotalIts = 0
 
+    ! we need to re-set the orders converged to 16 as it might have been modified in the previous iteration
+    ordersConverged = 16.0_realType
```

Comment:
Why is this necessary exactly? The ordersConverged gets set unconditionally below anyway.